c2corg_ui/static/partials/cards/articles.html

```diff
@@ -0,0 +1,20 @@
+<a ng-href="{{cardCtrl.createURL()}}" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
+  <div class="outing-author text-center">
+    <div class="outing-date">{{ doc['date_end'] | date: 'dd/MM/yyyy'}}</div>
+    <span class="icon-user"></span><br>
+    <b class="author-name">{{doc['author']['name']}}</b>
```

Comment:
I don't think those attributes are useful for article cards: date, author (especially for collaborative articles, that may have several authors).

The attributes you have added below seem OK to me (title, lang, quality, activities). Perhaps "article_type"?
Is there a document or an issue where the client would have listed the attributes for article cards?